config.yaml

```diff
@@ -124,6 +124,34 @@ languages:
           weight: 170
           parent: customnav_agentnav_basic_usage
 
+        ## TRACING/APM - Setup basic
+        - identifier: customnav_tracingnav_setup
+          name: "APM"
+          url: "agent/apm/"
+          weight: 13
+          parent: menu_references_agent
+
+        - identifier: docker_ecs
+          name: "Tracing ECS application"
```

Comment:
Is ECS different enough from other container environments to warrant its own section?

Per Amazon guidelines:
Amazon Elastic Container Service
or 
Amazon ECS